modules/drivers/redshift/test/metabase/driver/redshift_test.clj

```diff
@@ -480,3 +484,96 @@
                        qual-view-name
                        qual-mview-name
                        username)))))))))
+
+;;;; Server side generated timestamps for :relative-datetime tests follow.
+
+(defn- run-native-query [sql & params]
+  (-> (mt/native-query {:query sql
+                        :params params})
+      qp/process-query))
+
+(defn- getdate-vs-ss-ts-test-thunk-generator
+  ([]
+   (getdate-vs-ss-ts-test-thunk-generator :week -1))
```

Comment:
This is not a very intuitive default, I'm not sure it's worth defining it.

Reply:
That's right. I've removed zero arity and added arguments to calls where it was used. Thanks.